roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -15,15 +16,28 @@ class DockerStorage(DockerHostMixin, OpenShiftCheck):
 
     name = "docker_storage"
     tags = ["pre-install", "health", "preflight"]
+    # TODO(lmeyer): mention these in the output when check fails
+    configuration_variables = [
+        (
+            "max_thinpool_data_usage_percent",
+            "For 'devicemapper' storage driver, usage threshold percentage for data. Format: float. Default: 90.0",
+        ),
+        (
+            "max_thinpool_meta_usage_percent",
+            "For 'devicemapper' storage driver, usage threshold percentage for metadata. Format: float. Default: 90.0",
+        ),
+        (
+            "max_overlay_usage_percent",
+            "For 'overlay' or 'overlay2' storage driver, usage threshold percentage. Format: float. Default: 90.0",
```

Comment:
Suggestion: If we define `configuration_variables` below `max_overlay_usage_percent` et al, we could reference the class attributes instead of hardcoding the defaults / duplicating.

```python
In [1]: class Foo:
   ...:     bar = 1
   ...:     doc = 'Default bar: {}'.format(bar)
   ...:     

In [2]: Foo.doc
Out[2]: 'Default bar: 1'
```